lib/ui/lerp.dart

```diff
@@ -10,5 +10,5 @@ double lerpDouble(num a, num b, double t) {
     return null;
   a ??= 0.0;
   b ??= 0.0;
-  return a + (b - a) * t;
+  return a + (b - a) * t as double;
```

Comment:
Maybe some extra parens so I don't have to think about precedence?
```suggestion
  return (a + (b - a) * t) as double;
```

Reply:
I did it this way at first but it triggered the unnecessary parens lint.

Reply:
Well, that's too bad.  Almost seems like a bug in that lint: sure, it's clearly not strictly necessary, but neither is it clear to me that the `as double` applies to the whole thing as opposed to the `t` alone (which is already a double).

Anyhow, it's not worth putting in an "ignore:" so I'm fine with just leaving the parens off in this case.